drivers/hmis/spec/models/hmis/hud/enrollment_spec.rb

```diff
@@ -18,6 +18,49 @@
 
   include_context 'hmis base setup'
 
+  it 'detects date conflicts' do
+    [
+      # [ enter, exit, cmp enter, cmp exit, conflict expected ]
+      ['2000-01-01', '2000-01-01', '2000-01-01', nil, true, 'enter on exited entry date'],
+      ['2000-01-01', '2000-01-03', '2000-01-02', nil, true, 'enter between exited entry and exit'],
+      ['2000-01-02', '2000-01-03', '2000-01-01', nil, true, 'enter before existing entry'],
+      ['2000-01-01', nil,          '2000-01-02', nil, true, 'enter after another active'],
+      ['2000-01-01', '2000-01-02', '2000-01-02', nil, false, 'enter on exited exit date'],
```

Comment:
Entry should be allowed on the exit date. At least I think it should.  If you enter an Emergency Shelter, stay one night, leave.  You'd be exited on the next morning.  If you then showed up later that night, you might get another entry.  In an ideal world your previous enrollment would be re-opened and you'd just get a bed-night, but I don't think it should be prevented.

Reply:
Oh, I think this is indicating this is NOT an error.  Looks good.